packages/framework/src/Controller/Admin/DefaultController.php

```diff
@@ -160,6 +166,48 @@ protected function addWarningMessagesOnDashboard(): void
                 ],
             );
         }
+
+        $notSetTermsAndConditionsArticles = $this->legalConditionsFacade->findNullSettingValues(Setting::TERMS_AND_CONDITIONS_ARTICLE_ID);
+
+        foreach ($notSetTermsAndConditionsArticles as $notSetTermsAndConditions) {
+            $domain = $this->domain->getDomainConfigById($notSetTermsAndConditions->getDomainId());
+
+            $this->addErrorFlashTwig(
+                t('<a href="{{ url }}">Terms and conditions article for domain {{ domainName}} is not set.</a>'),
+                [
+                    'url' => $this->generateUrl('admin_legalconditions_termsandconditions'),
```

Comment:
If the article is not set on several domains, the link will be always the same because the domain can be switched only globally. 
I don't think that's a big problem, just wanted to point it out.

Reply:
Noticed that. Should i do something with that or is it fine to keep it as it is?

Reply:
Let's keep it as it is... I think it's ok :)